packages/squidex/src/entities/research-output.ts

```diff
@@ -12,6 +17,15 @@ export interface ResearchOutput {
   publishDate?: string;
   tags?: string[];
   adminNotes?: string;
+  subtype?: ResearchOutputSubtype;
+  accessInstructions?: string;
+  sharingStatus: ResearchOutputSharingStatus;
+  asapFunded: DecisionOption;
+  usedInAPublication: DecisionOption;
+  externalAuthors?: {
```

Comment:
i see how `tags` are declared in the same way - ie an optional property containing an array - but wouldn't it be easier to make it compulsory and keep it empty when there are no authors? Again, just asking. It's generally annoying when arrays are optional, nullable and can also be empty (although here null is not possible)